src/Model/Raster.php

```diff
@@ -33,26 +33,41 @@ public function getFilename()
 
     public function getSrid()
     {
+        $output = `gdalsrsinfo --version && cat /etc/issue`;
```

Comment:
This line should probably be removed.  But depending on which version of gdal is being used the output is different, I'm not sure of the best approach to fix this.  All I can think of is checking the version number, and tweaking call appropriately.  I've not double checked as I am typing this, but I think GitHub actions allows for older versions of Ubuntu, and this would allow unit testing of any said tweak.  See https://github.com/smindel/silverstripe-gis/issues/31